docs/browsers-api/index.md

```diff
@@ -14,18 +14,36 @@ Use `npx` to run the CLI:
 npx @puppeteer/browsers --help
 ```
 
-CLI help will provide all documentation you need to use the CLI.
+Built-in per-command `help` will provide all documentation you need to use the CLI.
 
 ```bash
 npx @puppeteer/browsers --help # help for all commands
 npx @puppeteer/browsers install --help # help for the install command
 npx @puppeteer/browsers launch --help # help for the launch command
 ```
 
+Some example to give an idea of what the CLI looks like (use the `--help` command for more examples):
+
+```sh
```

Comment:
The website prefers use of `bash` it renders the command better visually. 
```suggestion
```bash
```